frontend/src/components/cell/cell-text.tsx

```
@@ -5,21 +5,24 @@
+    <div className="deviceCell-textWrapper">
+      <CellHistory cell={cell} />
+
+      {cell.value && cell.readOnly && (
         <Tooltip
           text={<span><b>'{cell.value}'</b> {t('widgets.labels.copy')}</span>}
           placement="top-end"
           trigger="click"
         >
-          <div className="deviceCell-text" onClick={() => copyToClipboard(cell.value)}>
-            {cell.getEnumName(cell.value)}
+          <div className="deviceCell-text" onClick={() => copyToClipboard(cell.value as string)}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)